homeassistant/util/logging.py

```diff
@@ -62,6 +63,17 @@ def handle(self, record: logging.LogRecord) -> Any:
             self.emit(record)
         return return_value
 
+    def close(self) -> None:
```

Comment:
The goal there was to restore the original handlers since we were worried about messages in the queue not being written to the log at shutdown but it seems like the close you added is taking care of that?

Reply:
I think it's enough since the `close` is called from the logging at atexit handler, which is called very late: https://docs.python.org/3/library/atexit.html